src/commands/autocomplete/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import Command from 'cli-engine-command'
+import path from 'path'
+import {CustomColors} from 'cli-engine-command/lib/output'
+
+export class AutocompleteBase extends Command {
+  get autocompletePath (): string {
+    return path.join(this.config.dataDir, 'client', 'node_modules', 'cli-engine', 'autocomplete')
```

Comment:
@RasPhilCo not today, but we need to think about how this would work with CLI versions that do not auto-update or what happens if they have not had an update since installing